app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.includes(:contestants).find{params[:id]}
+    @challenge = @project.challenge
+    @theme = @challenge.theme
```

Comment:
This works just fine, but is a bit overkill because we already have access to challenge and theme data just from the `project` instance variable we're setting up in line 3